### PR TITLE
Add `migration` function for `DomainMigration`

### LINKED ISSUE #98: Update DomainMigration instances
We need to call update on the migration instance for domain migration instances

### PR DESCRIPTION
**Pull request by bengerman13**
## Changes proposed in this pull request:

- add `migration` function for `DomainMigration` fixes #98 
- add active certificate logic to `DomainMigration`
- move `migration` function to `Migration`, wrapping
  `DomainMigration::_migration` an `CdnMigration::_migration`
- add govcloud iam config

## Security considerations

None

### CODE REVIEW

**Comment by bengerman13** [comment]
I am very sorry this got so big - it should've been smaller chunks. 

Feel free to reach out if you want to pair on it for review :)

File: ci/pipeline.yml [hunk 1]
```diff
@@ -120,6 +123,9 @@ jobs:
         ENV: staging
         DNS_ROOT_DOMAIN: ((staging-dns-root-domain))
         DATABASE_ENCRYPTION_KEY: ((staging-database-encryption-key))
+        AWS_GOVCLOUD_REGION: ((staging-aws-govcloud-region))
+        AWS_GOVCLOUD_ACCESS_KEY_ID: ((staging-aws-govcloud-access-key-id))
+        AWS_GOVCLOUD_SECRET_ACCESS_KEY: ((staging-aws-govcloud-secret-access-key))
```
Comment (ccostino) [inline ci/pipeline.yml:128]: Is there a need for `production` versions of these environment variables, or not yet because this is still being tested?

**Review by ccostino — APPROVED** [review]
Thanks, @bengerman13!  This all looks fine to me; one quick question but not something to hold up moving forward!

File: ci/pipeline.yml (hunk 1 above)
Comment (bengerman13) [reply ci/pipeline.yml:128]: not yet - we haven't deployed this to prod yet

**Pull request merged**